mne/beamformer/_rap_music.py

```diff
@@ -31,6 +33,18 @@ def _apply_rap_music(data, info, times, forward, noise_cov, n_dipoles=2,
         Forward operator.
     noise_cov : instance of Covariance
         The noise covariance.
+    nave : int
+        Number of averages.
+    aspect_kind : int
+        The kind of data.
+    first : int
+        First sample.
+    last : int
+        Last sample.
```

Comment:
can't you infer this from the times and info['sfreq'] ?

I think one needs to add this param.